### PR TITLE
Variables and data - Winton Fong

### PR DESCRIPTION
**Pull request by wintonsec**
Screenshot 2024-07-17 at 17 50 05

### CODE REVIEW

File: variables-and-data/quiz.md
Comment (AcquirEdge) [inline variables-and-data/quiz.md:1]: Need to complete the quiz questions

**Review by AcquirEdge — CHANGES_REQUESTED** [review]
Complete the questions in quiz.md!

File: variables-and-data/quiz.md
Comment (wintonsec) [reply variables-and-data/quiz.md:1]: resubmitted

File: variables-and-data/quiz.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Variables Quiz
+
+1. What does the `=` (equals) operator do in JavaScript?
+   It's used to assign a value to a variable
+2. What operator shows the data type of a JavaScript variable or value?
+   "typeof" operators return strings that indicate the type of the operand's value
+3. Which keyword is used to declare a variable in JavaScript?
+   variables in JavaScript can be declared using either 'var', 'let', or 'const' keywords
```
Comment (AcquirEdge) [inline variables-and-data/quiz.md:8]: In our program we'll only be using the "let" keyword, but absolutely correct.

**Review by AcquirEdge — APPROVED** [review]
Looks good to me!

**Pull request merged**